LayoutKitSampleApp/Benchmarks/FeedItemLayoutKitView.swift

```diff
@@ -24,13 +32,20 @@ class FeedItemLayoutKitView: UIView, DataBinder {
 
         let content = ContentLayout(title: data.contentTitle, domain: data.contentDomain)
         layout = FeedItemLayout(actionText: data.actionText, posterProfile: posterProfile, posterComment: data.posterComment, contentLayout: content, actorComment: data.actorComment)
+        
+        setNeedsLayout()
+        
+        if intrinsicContentSizeCalculated {
```

Comment:
do you really need to wait for intrinsicContentSize to be called or do you just want bounds.width to be non-zero?

```
if bounds.width > 0 {
    heightConstraint.constant = sizeThatFits(CGSize(width: bounds.width, height: .greatestFiniteMagnitude)).height
}

```